src/interface/common.thrift

```diff
@@ -55,6 +57,28 @@ enum SupportedType {
     // STRUCT = 104,
 } (cpp.enum_strict)
 
+enum IndexType {
+    UNKNOWN = 0,
+    TAG = 1,
+    EDGE = 2,
+}
+
+enum IndexOpType {
```

Comment:
I think we should process the index when we modify the data. So this is unnecessary.

Reply:
How to process data modify during the index creating ? I think ，we need action log to redo the data modify actions after the original data is assembled.
Data modification occurs in two scenarios ：1, index is NORMAL status . 2, index is BUILDING
When index is building, we need save all data modify log . the log prefix is '_wal_i'